localstripe/resources.py

```diff
@@ -944,6 +944,25 @@ def _api_list_subscriptions(cls, id, **kwargs):
 
         return cls._api_retrieve(id).subscriptions
 
+    @classmethod
+    def _api_list_sources(cls, id, object=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
```

Comment:
Maybe this check should be in your other commits too?

Reply:
i think localstripe should be less pedantic about missing query parameters, especially given that the APIs are not implemented completely. this is also why i did not include the check everywhere. other existing methods also don't do the check.

Reply:
I agree with @H--o-l, this pedantic error message is quite helpful, and I'm wondering if Stripe also does it. Since most methods do the check, I vote to put it in new ones we add.

Reply:
ok. this will however force me to keep using our fork.

Reply:
i'll quickly test this with stripe. will get back to you